packages/catalog-client/src/types.ts

```diff
@@ -16,10 +16,17 @@
 
 import { Entity, EntityName, Location } from '@backstage/catalog-model';
 
+export type CatalogEntitiesRequest =
+  | {
+      filter?: Record<string, string | string[]> | undefined;
+      fields?: string[] | undefined;
+    }
+  | undefined;
```

Comment:
Sup with all the `undefined`s here when they're already optional?

Reply:
removed